torch/jit/__init__.py

```diff
@@ -728,8 +731,9 @@ def weighted_kernel_sum(self, weight):
         example_weight = torch.rand(1, 1, 3, 3)
         example_forward_input = torch.rand(1, 1, 3, 3)
         n = Net()
-        inputs = {'forward' : example_forward_input, 'weighted_kernel_sum' : example_weight}
-        module = torch.jit.trace_module(n, inputs)
+        # the following two calls are equivalent
+        module = torch.jit.trace_module(n, example_forward_input)
+        module = torch.jit.trace_module(n.forward, example_forward_input)
```

Comment:
Uh, do we really believe that people will want to `n.forward`?

Reply:
I think we saw a few examples of people using it.